diff --git a/setup.py b/setup.py
index 75c6ec3..aa84d0a 100644
--- a/setup.py
+++ b/setup.py
@@ -27,7 +27,7 @@ setup(
     long_description=readme,
     long_description_content_type="text/markdown",
     python_requires=">=3.6",
-    packages=find_packages(exclude=["docs", ".gitignore", "README.md"]),
+    packages=find_packages(exclude=["tests*", "docs", ".gitignore", "README.md"]),
     install_requires=required,
     classifiers=[
         "Environment :: Console",
